addons/im_livechat/static/src/js/im_livechat.js

```diff
@@ -328,7 +328,11 @@ var LivechatButton = Widget.extend({
         var self = this;
         return session
             .rpc('/mail/chat_post', {uuid: this._livechat.getUUID(), message_content: message.content})
-            .then(function () {
+            .then(function (result) {
+                if (!result) {
+                    alert(_t("You took to long to send a message. \nPlease refresh the page and try again."));
```

Comment:
alert ?

Pretty sure using "alert" in 2019 is an automatic 🚩

(More seriously, please update to displayNotification instead)